Docs/ZynqVitisXpfm.md

```diff
@@ -0,0 +1,41 @@
+# Building Vitis XPFM platform for Hastlayer (Trenz TE0715-04-30-1C module)
+
+To be able to generate Vitis XCLBIN files (containing the Hastlayer RTL kernel) for Trenz Electronics TE0715-04-30-1C Zynq 7030 SOM SoC module you have to generate the Vitis XPFM platform based on the prevously generated Vivado XSA file. This document describes the required steps.
+ 
+Before using the Vitis tools you first have to set up the environment:
+ 
+```
+source /tools/Xilinx/Vitis/2020.2/settings64.sh
+```
+
+Create an empty folder and launch Vitis GUI:
+
+```
+mkdir ${HOME}/trenz_te0715_04_30_1c/vitis
+cd ${HOME}/trenz_te0715_04_30_1c/vitis
+vitis
+```
+
+Set Workspace to: ${HOME}/trenz_te0715_04_30_1c/vitis
+
+Create a new platform project:
+
+File - New - Platform Project...
+
+![ZynqVitisXpfmNewPlatformProject](Images/ZynqVitisXpfmNewPlatformProject.png)
+
+Platform project name: trenz_te0715_04_30_1c
+
+XSA File: ${HOME}/trenz_te0715_04_30_1c/vivado/test_board/vivado/zsys_wrapper.xsa
+
+Uncheck "Generate boot components"
```

Comment:
Ez hol van? Nekem nem jelenik meg ilyen opció.
![image](https://user-images.githubusercontent.com/4007293/153724540-1ced8e5c-134a-4bab-be49-8445496d118b.png)

És ha nyomok egy Finish-t akkor ezt írja:
![image](https://user-images.githubusercontent.com/4007293/153724572-cbc101e6-740f-4199-930e-2705a2a94856.png)

Reply:
![image](https://user-images.githubusercontent.com/95757648/154332858-4bc1fde8-8ac0-4c18-93df-1cf439d2b0a0.png)
Nálam ez van. Mivel nálad nincs ott az OS és a CPU típusa sem, ezért elsőre az a tippem, hogy a Vivado-ban valami beállítás kimaradhatott.